.github/workflows/java-ci-latest.yml

```diff
@@ -18,6 +18,7 @@ on:
   push:
     branches:
       - main
+  pull_request: {} # WIP
```

Comment:
- [ ] Remove before merge, we only need awareness in main

Reply:
```suggestion
```

The latest build passed [here](https://github.com/SDA-SE/sda-spring-boot-commons/actions/runs/13765334941/job/38490350040?pr=875).